JapaneseFilter/sections/specific.txt

```diff
@@ -1998,10 +1995,23 @@ intensive911.com##div[id^="adx_"]
 clicccar.com##.ad-management
 quick-timez.com##.k1y-inline-ad
 ferret-plus.com##div[class*="-ad-banner"]
-itmedia.co.jp###ITN
+/ad.js$domain=itmedia.co.jp
+||itmedia.co.jp^*/file_bit_article_bottom_ad.html
+||itmedia.co.jp/css/sa.js
+||itmedia.co.jp/css/sa_ini.js
 itmedia.co.jp###ATF
+itmedia.co.jp###ITN
+itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]
```

Comment:
On my end this doesn't hit. Do you see `#ITT`?

Reply:
I just move the rule,so I am not sure if the rule is still work.
Do you want to delete it?
These rules are added from this commit.
`itmedia.co.jp###ITN`
https://github.com/AdguardTeam/AdguardFilters/commit/c8c9c7028b1ab9d6b97a9d5768f5c1bfbaf87fb8
`itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]`
https://github.com/AdguardTeam/AdguardFilters/commit/ff64356b2035f6ea4c9a4f56be366581f6d6678a

Reply:
I can see `#ITT` in issue address
<details><summary>screenshot</summary>

![](https://i.postimg.cc/sXsNhV8Q/image.png)

</details><br/>

Reply:
Oh, I mistook. I still se the requests on the issue URL so no need to remove.

Reply:
If `#ITT` covers the target, it'll be better.

Reply:
`#ITT` is PR article.
Should I make a new rule as self-promo?

Reply:
You're right, sorry it seems I can't do anything right today. Apparently `###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]` is obsolete now so can be removed.

Reply:
On my end those PR leads to unrelated third-party sites. If this is true to you too, they can be addressed in JPF.

Reply:
I see. I will add rule of self-promo and remove `###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]` . Thank you for review amd advice.